### ISSUE TITLE
📦 New Logo for WPGulp

### ISSUE THREAD

**Issue by ahmadawais** [ahmadawais/WPGulp#62]
The task here is to create a new LOGO for the new WPGulp version 2.0.0. 

- [ ] Must be red (Gulp is red)
- [ ] Different vertical and horizontal sizes
- [ ] Featured images or press kit for blogs to share

This is one of the tasks from our goals #61 for WPGulp v2.0.0
Peace! ✌️

**Comment by JeremyEnglert** [comment]
@ahmadawais - do you have someone working on this? If not, I may know a few designers interested in contributing!

**Comment by dmhendricks** [comment]
I, too, can try to put some thing(s) together if you give me an idea of what you want it to look like. That is, if you're looking for help/ideas.

**Comment by ahmadawais** [comment]
@JeremyEnglert Sure! Go ahead and spread the word.
@dmhendricks sure take a stab at it. We had this thing before. I was thinking of improving it in this iteration.

![image](https://pbs.twimg.com/media/CNIpbI1UcAA72sS.jpg:large)

**Comment by ahmadawais** [comment]
@dmhendricks @JeremyEnglert any updates on this one?

**Comment by JeremyEnglert** [comment]
I know a designer interested in helping, but he's swamped for the next couple of weeks. However, he's willing to jump on then if needed!

**Comment by AndreaBarghigiani** [comment]
I really love this project and I had this strange idea about the new logo, I understand it is really different from the starting point but I thought that the image you show us is more than a background than a logo. So this is my first try and everyone can use it as a starting point if they like it.

![wplogo-test](https://user-images.githubusercontent.com/190159/34665141-6a15dc66-f45f-11e7-85d6-3ac5a8942783.png)

**Comment by ahmadawais** [comment]
@AndreaBarghigiani Thanks a lot for your effort but, I think we need something quite minimal and simple. Nothing too fancy or creative. 👍

**Comment by theenoahmason** [comment]
I am a designer, I am about to give this package a go. If It suits my needs, I will offer a small identity package as a thank you.

**Comment by ahmadawais** [comment]
That's awesome, Noah! Do checkout the latest v2.0.0 branch. We are close to
releasing a completely new zero configuration WPGulp package.





*  AHMAD AWAIS  *
  Full Stack Web Developer — Regular WordPress Core Contributor/Dev

*AhmadAwais.com
<https://ahmadawais.com/?utm_source=email_signature&utm_medium=email>** |
Say Hi on Twitter <https://twitter.com/mrahmadawais/>* *| **Facebook
<https://fb.com/ahmadawais/> **| **GitHub <https://github.com/ahmadawais> *
*| **WordPress <https://profiles.wordpress.org/mrahmadawais/> **|* *LinkedIn
<http://pk.linkedin.com/in/mrahmadawais>*
 *Skype/Slack: MrAhmadAwais *

**Comment by mrxkon** [comment]
I was talking with @ahmadawais today about a similar idea of this ( I just hadn't heard WPGulp yet it seemed :D ) and I just have to say 🥇.

Anyway I'm not a graphic designer per say but I have been on a similar field ( of tattooing ) for years now along with my love of coding so: The idea is that everybody already knows both `logos` very well top say the least and since I'm not the guy that tries to re-invent the wheel if it's already there for use

WP-Gulp could just be translated into 

![logo_wpgulp2_preview](https://user-images.githubusercontent.com/22611066/37028433-e9cc3220-213c-11e8-9110-2f583eb68a57.jpg)

SVG Preview [HERE](https://xkon.gr/files/logo_WPGulp2.svg).

SVG Code:
```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="522 0 500 500">
  <path fill="#E94B4C" d="M522 0h500v500H522V0z"/>
  <path fill="#CE3F41" d="M1022 500l-229.598-.002-67.199-67.199 108.224-81.309 4.019-7.141 20.393-183.425.814-7.27L1022 317v78z"/>
  <path fill="#FFF" d="M771.053 361.239c-22.574 0-40.017-3.392-48.405-5.412-5.736-1.383-10.342-2.812-13.841-4.282l11.187 19.114 4.376 59.719c0 6.694 21.268 12.122 47.503 12.122 26.234 0 47.503-5.428 47.503-12.122l3.465-60.368 10.586-18.52c-3.514 1.489-8.162 2.938-13.97 4.337-8.388 2.021-25.831 5.412-48.404 5.412m.82-218.983c-47.928 0-86.78 5.427-86.78 12.122s38.852 12.122 86.78 12.122 86.78-5.427 86.78-12.122-38.852-12.122-86.78-12.122m24.332 18.557c-5.711 0-10.34-1.261-10.34-2.816 0-1.556 4.629-2.817 10.34-2.817 5.71 0 10.341 1.261 10.341 2.817 0 1.555-4.631 2.816-10.341 2.816"/>
  <path fill="#FFF" d="M804.528 157.002c0 1.006-3.726 1.822-8.323 1.822s-8.323-.816-8.323-1.822 3.727-1.822 8.323-1.822c4.598.001 8.323.817 8.323 1.822m54.145-89.766c-.928 1.05-4.188-.333-7.268-3.08-3.081-2.747-5.183-5.539-4.197-6.536 1.034-1.046 4.36-.116 7.442 2.632 3.081 2.747 4.938 5.949 4.023 6.984"/>
  <path fill="#FFF" d="M858.673 67.236l-11.465-9.616-46.15 48.562-13.162 50.821h16.631l10.872-44.079z"/>
  <g fill="#FFF">
    <path d="M771.999 175.893c-13.837 0-25.095 11.257-25.095 25.094 0 13.838 11.258 25.096 25.095 25.096s25.095-11.258 25.095-25.096c0-13.837-11.258-25.094-25.095-25.094zm-22.562 25.094c0-3.271.701-6.376 1.953-9.182l10.763 29.489c-7.526-3.658-12.716-11.376-12.716-20.307zm22.562 22.564c-2.215 0-4.352-.325-6.374-.919l6.77-19.671 6.938 19c.045.112.1.214.16.311-2.347.825-4.866 1.279-7.494 1.279zm3.11-33.143c1.357-.071 2.582-.215 2.582-.215 1.217-.144 1.072-1.93-.145-1.859 0 0-3.654.287-6.014.287-2.217 0-5.943-.287-5.943-.287-1.216-.071-1.358 1.788-.143 1.859 0 0 1.151.144 2.367.215l3.515 9.632-4.938 14.808-8.216-24.438c1.36-.071 2.582-.216 2.582-.216 1.215-.144 1.072-1.93-.145-1.858 0 0-3.653.286-6.013.286-.424 0-.922-.01-1.452-.026 4.034-6.126 10.97-10.17 18.852-10.17 5.874 0 11.222 2.246 15.237 5.923-.098-.006-.193-.018-.293-.018-2.215 0-3.789 1.931-3.789 4.004 0 1.859 1.072 3.433 2.217 5.292.859 1.503 1.859 3.434 1.859 6.223 0 1.932-.742 4.173-1.717 7.294l-2.25 7.52-8.153-24.256zm16.688-.246c1.762 3.214 2.766 6.902 2.766 10.825 0 8.324-4.514 15.591-11.221 19.503l6.893-19.927c1.287-3.218 1.715-5.792 1.715-8.082-.001-.828-.055-1.6-.153-2.319z"/>
    <path d="M837.122 263.32c-1.037 1.561-2.158 3.078-3.361 4.555-1.205 1.477-2.332 2.82-3.383 4.031-1.051 1.213-1.934 2.241-2.648 3.086-.713.848-1.086 1.42-1.113 1.719 0 .398.127.721.379.971.252.248.672.373 1.26.373.477 0 1.178-.266 2.102-.797s2.221-1.451 3.887-2.762c1.666-1.312 3.783-3.07 6.346-5.277 1.758-1.512 3.816-3.314 6.135-5.367l11.115-102.928c-1.791.988-3.652 1.608-4.75 1.971-4.6 1.518-11.093 2.87-19.3 4.016-16.567 2.314-38.557 3.589-61.916 3.589-23.359 0-45.348-1.275-61.917-3.589-8.207-1.146-14.7-2.498-19.298-4.016-1.097-.363-2.956-.982-4.745-1.968l10.264 94.473c.478-1.627 1.009-3.252 1.598-4.871 1.204-3.318 2.599-6.563 4.181-9.731 1.583-3.168 3.305-6.163 5.169-8.983 1.862-2.82 3.844-5.417 5.946-7.791 2.101-2.372 4.258-4.412 6.472-6.122 2.212-1.708 4.447-3.044 6.702-4.007 2.256-.961 4.49-1.443 6.703-1.443 2.549 0 5.014.697 7.396 2.091 2.381 1.393 4.608 3.65 6.682 6.769.475.697.777 1.402.902 2.115.127.714.19 1.386.19 2.016 0 1.493-.358 2.672-1.071 3.534-.714.863-1.562 1.294-2.542 1.294-.898 0-1.731-.473-2.501-1.419-.771-.945-1.632-2.297-2.586-4.056-.896-1.593-1.883-2.745-2.961-3.459-1.079-.714-2.248-1.07-3.509-1.07-1.961 0-4.055.813-6.282 2.439-2.228 1.626-4.434 3.774-6.619 6.445-2.185 2.672-4.28 5.724-6.283 9.16-2.003 3.434-3.775 6.951-5.316 10.551-1.541 3.602-2.766 7.152-3.676 10.652-.911 3.5-1.366 6.645-1.366 9.432 0 .93.063 1.885.188 2.861.127.98.365 1.867.715 2.664.35.797.826 1.451 1.429 1.964.602.517 1.366.772 2.29.772 1.037 0 2.213-.348 3.53-1.045 1.317-.695 2.683-1.609 4.097-2.736 1.415-1.129 2.837-2.407 4.266-3.834 1.429-1.426 2.759-2.869 3.992-4.33 1.232-1.459 2.325-2.861 3.277-4.205.953-1.344 1.667-2.514 2.144-3.51l3.908-11.697c.475-1.26 1.098-2.156 1.87-2.687.769-.531 1.561-.797 2.375-.797.391 0 .783.067 1.176.198.392.134.742.342 1.05.623.309.283.553.646.736 1.096.181.447.272.97.272 1.566 0 1.594-.139 3.086-.42 4.48-.279 1.395-.636 2.754-1.071 4.082-.434 1.326-.918 2.662-1.45 4.006-.532 1.344-1.051 2.746-1.555 4.207-.111.166-.314.688-.609 1.566-.294.881-.651 1.984-1.072 3.311-.419 1.328-.876 2.813-1.366 4.455-.49 1.643-.994 3.318-1.511 5.027-.52 1.709-1.016 3.385-1.493 5.027-.476 1.643-.904 3.127-1.281 4.455-.379 1.326-.693 2.438-.946 3.334-.252.896-.406 1.426-.462 1.593-.141.53-.365 1.12-.672 1.767-.308.646-.673 1.262-1.094 1.842-.42.58-.882 1.063-1.386 1.443-.504.381-1.023.572-1.555.572-.897 0-1.548-.299-1.953-.895-.407-.598-.61-1.693-.61-3.285v-.697c0-.232.014-.482.041-.747.028-.53.239-1.526.631-2.987.393-1.459.862-3.109 1.409-4.951.545-1.842 1.134-3.75 1.764-5.725.63-1.975 1.197-3.74 1.702-5.301-1.486 1.66-3.095 3.311-4.833 4.953-1.737 1.643-3.516 3.119-5.337 4.43-1.821 1.311-3.656 2.381-5.505 3.211-1.848.828-3.627 1.244-5.336 1.244-1.961 0-3.65-.432-5.065-1.295-.256-.156-.496-.332-.736-.506l5.712 59.889c1.486 3.727 29.48 12.884 66.39 12.884 36.928 0 64.932-9.166 66.394-12.891l8.205-71.201c-.68.621-1.34 1.231-1.973 1.821-2.354 2.189-4.574 4.072-6.662 5.648-2.086 1.576-4.125 2.797-6.113 3.658-1.99.863-4.119 1.295-6.389 1.295-2.129 0-3.775-.465-4.938-1.395-1.162-.928-1.744-2.205-1.744-3.832v-.373c0-.115.014-.258.043-.424.111-.994.547-2.141 1.303-3.434.756-1.295 1.666-2.639 2.73-4.031 1.064-1.395 2.199-2.805 3.404-4.232 1.205-1.426 2.318-2.777 3.342-4.057 1.021-1.275 1.869-2.445 2.541-3.508.674-1.063 1.01-1.924 1.01-2.588 0-.398-.141-.713-.42-.947-.281-.23-.744-.348-1.387-.348-1.121 0-2.277.291-3.467.871-1.191.58-2.369 1.361-3.531 2.34s-2.297 2.1-3.404 3.359c-1.105 1.262-2.143 2.572-3.109 3.932-.967 1.361-1.848 2.73-2.646 4.107s-1.479 2.646-2.039 3.807c-.168.365-.406.881-.715 1.543-.309.664-.645 1.412-1.008 2.24-.365.83-.75 1.686-1.156 2.563-.406.881-.777 1.711-1.113 2.49s-.637 1.475-.902 2.09c-.268.613-.457 1.037-.568 1.27-.141.299-.371.861-.693 1.691s-.693 1.801-1.113 2.912-.861 2.281-1.324 3.51c-.461 1.227-.897 2.389-1.303 3.484-.406 1.094-.764 2.057-1.07 2.887-.309.828-.506 1.393-.59 1.692-.279.862-.609 1.674-.986 2.439-.379.762-.813 1.434-1.303 2.015s-1.051 1.046-1.682 1.394c-.632.35-1.339.522-2.121.522-1.011 0-1.843-.241-2.502-.722-.656-.48-.986-1.403-.986-2.762 0-.664.084-1.344.252-2.04.168-.697.379-1.387.631-2.066.252-.682.512-1.344.777-1.99.266-.648.512-1.27.734-1.867 1.514-3.318 3.055-6.645 4.623-9.98 1.568-3.334 3.074-6.635 4.518-9.904.301-.682.576-1.35.869-2.027-.088.086-.174.176-.26.26-1.822 1.76-3.635 3.312-5.441 4.654-1.809 1.344-3.602 2.414-5.379 3.211-1.779.795-3.51 1.192-5.191 1.192-1.148 0-2.107-.181-2.877-.546-.771-.365-1.387-.863-1.85-1.492-.463-.631-.793-1.377-.988-2.24-.066-.295-.107-.604-.15-.91-.492.383-.988.756-1.488 1.11-1.736 1.226-3.445 2.214-5.127 2.96-1.68.747-3.193 1.118-4.537 1.118-1.346 0-2.438-.446-3.279-1.343-.84-.895-1.26-2.422-1.26-4.578 0-1.527.225-3.303.672-5.325-.729 1.294-1.639 2.604-2.73 3.931-1.094 1.328-2.34 2.539-3.741 3.635-1.402 1.094-2.97 1.982-4.707 2.662-1.738.68-3.629 1.019-5.673 1.019-.924 0-1.813-.114-2.669-.347-.854-.231-1.603-.631-2.247-1.195-.645-.563-1.163-1.318-1.555-2.264-.393-.945-.588-2.133-.588-3.559 0-.033.056-.539.168-1.52.111-.979.469-2.521 1.071-4.629.603-2.105 1.534-4.836 2.794-8.188 1.262-3.352 3.04-7.416 5.338-12.193.588-1.262 1.268-2.166 2.037-2.713.772-.548 1.604-.822 2.5-.822.393 0 .799.059 1.219.174.422.117.812.299 1.178.549.363.248.658.564.882.945.223.381.336.82.336 1.318 0 .332-.043.648-.127.945-.14.566-.441 1.303-.902 2.215-.463.914-1.002 1.959-1.619 3.137-.615 1.178-1.268 2.447-1.953 3.809-.688 1.359-1.324 2.77-1.912 4.23-.588 1.461-1.093 2.945-1.513 4.455-.421 1.51-.659 2.979-.714 4.405 0 .597.139 1.144.42 1.642.279.498.742.746 1.387.746 1.82 0 3.556-.598 5.211-1.791 1.652-1.195 3.177-2.679 4.58-4.455 1.4-1.775 2.668-3.684 3.802-5.724 1.136-2.042 2.081-3.907 2.837-5.601.533-1.127 1.002-2.34 1.408-3.633.406-1.295.848-2.488 1.324-3.584s1.021-2.008 1.638-2.738c.616-.729 1.388-1.095 2.312-1.095 1.01 0 1.807.382 2.396 1.146.588.764.881 1.676.881 2.736 0 .598-.188 1.42-.566 2.465s-.855 2.24-1.43 3.584-1.197 2.779-1.869 4.305c-.672 1.527-1.297 3.078-1.869 4.654-.576 1.576-1.051 3.119-1.43 4.629s-.566 2.912-.566 4.205c0 1.195.615 1.793 1.848 1.793.869 0 1.947-.299 3.236-.896s2.641-1.436 4.055-2.513c.354-.271.709-.552 1.063-.847.395-1.629.857-3.135 1.396-4.504.98-2.59 1.975-5.16 2.984-7.715 1.008-2.555 1.932-4.895 2.773-7.02l12.732-32.104c.504-1.294 1.184-2.207 2.037-2.738.855-.53 1.717-.796 2.586-.796.867 0 1.658.282 2.373.846.715.565 1.072 1.477 1.072 2.738 0 .597-.119 1.228-.357 1.891-.239.665-.539 1.377-.904 2.14-.699 1.561-1.533 3.468-2.5 5.724-.967 2.258-2.004 4.713-3.109 7.367-1.107 2.655-2.242 5.417-3.404 8.287-1.162 2.871-2.297 5.666-3.404 8.387-1.105 2.723-2.143 5.301-3.109 7.74s-1.799 4.555-2.5 6.346c-.309.764-.609 1.652-.902 2.664-.295 1.012-.441 1.916-.441 2.713 0 .496.084.895.252 1.193s.49.449.967.449c.588 0 1.322-.207 2.205-.623.883-.414 1.834-.963 2.857-1.644 1.023-.679 2.087-1.459 3.193-2.339 1.107-.879 2.207-1.791 3.299-2.736 1.094-.947 2.137-1.893 3.131-2.838s1.885-1.816 2.67-2.613c.184-.191.404-.326.646-.427.473-1.413.904-2.798 1.285-4.151.225-.598.533-1.443.926-2.539.391-1.096.84-2.174 1.344-3.236.504-1.061 1.072-1.982 1.701-2.762.631-.779 1.31-1.17 2.039-1.17 1.121 0 1.934.258 2.438.771.504.516.756 1.27.756 2.266 0 .199-.027.514-.084.945s-.125.889-.209 1.369-.176.938-.273 1.369c-.1.432-.189.746-.274.945 1.036-1.129 2.157-2.248 3.362-3.359s2.451-2.117 3.74-3.012c1.289-.896 2.605-1.625 3.951-2.191 1.344-.563 2.689-.846 4.033-.846.924 0 1.842.133 2.752.399.91.265 1.725.655 2.438 1.169.715.516 1.297 1.162 1.744 1.941s.672 1.684.672 2.713c0 1.361-.342 2.813-1.029 4.355-.693 1.55-1.555 3.101-2.592 4.659zm-65.123-36.031c-14.502 0-26.301-11.798-26.301-26.302 0-14.501 11.799-26.3 26.301-26.3 14.503 0 26.302 11.798 26.304 26.3 0 14.504-11.801 26.302-26.304 26.302z"/>
  </g>
</svg>
```

Zip contains both svg + ai (cs5).
[logo_WPGulp2.zip](https://github.com/ahmadawais/WPGulp/files/1784642/logo_WPGulp2.zip)

**Comment by ahmadawais** [comment]
@mrxkon That's very kind of you to say! 💯 and the logo. I'll def look at it when releasing v2.0.0.

**Comment by imomer** [comment]
Just a quicky work.... 

![wpglup](https://user-images.githubusercontent.com/773535/37096755-7e86e5d2-223b-11e8-9b42-97aa1ee48bfa.jpg)

**Comment by marianrick** [comment]
It may seem a little bit fussy, but Wordpress has got some clear guidelines on how to use their logo. Whenever you use the visual identity of another organization, I consider it appropriate to use their identity as it comes. Because of that I would avoid rotating or manipulating it. The Wordpress logo should stand tall & graceful at it is. 

Furthermore Wordpress has a clear opinion of their own inside their [Trademark Policy](http://wordpressfoundation.org/trademark-policy/):
> It’s OK to use the WordPress [...] logo as part of a page that describes your products or services, but it is not OK to use it as part of your company or product logo or branding itself.

Considering this I would recommend you to maybe use "WPGulp" as a name of its own, instead of just a combination of both of their logos, and add "Wordpress + Gulp" as a tagline to it?

**Although some ideas on how it should behave:**

- its accessible and fun to use
- it makes your life easier 
- its about achieving something fast and automated
- its lightweight, but does the heavy work
- the font should be free to use and accessible as well
- the colors should combine Wordpress blue/orange and adapt gulps red
- it should not be too bold, as it will be used in small sizes more often
- its powered by the community, so it can be very diverse

These are general suggestions and they do not claim to be right nor complete.

**Comment by marianrick** [comment]
Its really hard to make a good logo, but I thought I cannot give these suggestions without at least trying it on my own. Maybe its a good starting point for another person.

**Input:**
![input](https://user-images.githubusercontent.com/20360130/37915564-1a4f37c0-311a-11e8-9f71-718affc745d0.png)
I thought its a good idea if the logo tries to keep both visual words (Wordpress and Gulp) alive, while it still refers to the associations given with "to gulp", used by the straw in the current logo (is this correct English?!).

As the Wordpress is slightly more orange and the gulp appears to be slightly more red, the blue is the color that distinguishes. So this can be a great connector between both visual worlds. 

Although straws have something playful, something fun. Maybe we can use this, to make WPGulp more fun to watch and fun to use?

Long story short, this could be a logo, trying to achieve all of what has been said:
![output](https://user-images.githubusercontent.com/20360130/37915872-c831d276-311a-11e8-994f-74f06558fa53.png)
![variant](https://user-images.githubusercontent.com/20360130/37915911-ccf70268-311a-11e8-8563-f3f8d310b394.png)
![output with tagline](https://user-images.githubusercontent.com/20360130/37915923-d092dfb4-311a-11e8-96ba-e059aa7fee2f.png)

Of course this one is open for feedback and critics. Do you love it or hate it? If you are not into talking, but about doing, [here is the sketch file of this draft](https://www.file-upload.net/download-13051221/WPGulp-Sketch-MR-180326.sketch.html).

**Comment by ahmadawais** [comment]
💯 @marianrick I agree with everything you said.
The earlier image was just a featured image for a blog post I wrote, that had both WordPress and Gulp.js logos on it. 

✅ Also, I see that you put your money where your mouth is. I like the iterations of your logo designs and I specifically like this one.

![image](https://user-images.githubusercontent.com/20360130/37915923-d092dfb4-311a-11e8-96ba-e059aa7fee2f.png)

Can we play around with the font of the word WPGULP? I'd be happy to go with this version. But feels like something is missing. 

Looking forward to your new iterations and welcome as a contributor here.
Peace! ✌️

**Comment by marianrick** [comment]
@ahmadawais glad you like it, and of course I am willing to adjust the typeface. But as this is certainly a wide field, please try to give me a hint, what you feel like missing.

This would make much more sense than trying six different random typefaces. As its easier to disagree on something than to agree, here are some statements, you can choose out of or use them to make your own.

- It should look more lightweight
- It should distinguish itself more from other fonts
- Its too soft and round, can it be more technical? 
- I would like it to bee much bolder 
- It feels like its competes with the typeface of "Gulp"
- Can it look more natural

Although its super helpful to provide examples of a typeface you like, which do not miss out. For example "The Red Bull Logo looks neat" or something like that.

In the end, the last 20% will always depend on your personal taste. And as you are the dynamo of this repo, I would love to match it to yours.

**Comment by ahmadawais** [comment]
@marianrick I'm really not sure here. I like the direction but the typeface is a bit kid-like. I think we can have a more bold font face. I'll roll in @richtabor and let's what he thinks. ⚡

**Comment by richtabor** [comment]
@ahmadawais What do you think of something like this? 

![wpgulp](https://user-images.githubusercontent.com/1813435/38006291-6215d87c-3212-11e8-9ef5-c294093f117a.png)

**Comment by ahmadawais** [comment]
@richtabor I like that a lot. Can we play with the colors a bit like the ones @marianrick created? I think a mix of these two logos and we are good to go. 

To be clear, I like this logo ↓

![image](https://user-images.githubusercontent.com/1813435/38006291-6215d87c-3212-11e8-9ef5-c294093f117a.png)

But I'd love the above logo to have the following two colors. Maybe in that glass somewhere, or `WP` could be blue and `Gulp` could be red or something.

![image](https://user-images.githubusercontent.com/20360130/37915923-d092dfb4-311a-11e8-96ba-e059aa7fee2f.png)

**Comment by richtabor** [comment]
@ahmadawais Good idea. What do you think of this ↓
![wpgulp](https://user-images.githubusercontent.com/1813435/38025587-e56d6e94-3256-11e8-9cda-43d0cda959b4.png)

**Comment by ahmadawais** [comment]
@richtabor sold. Give me the `.psd` 👍 thanks for your design contribution.

**Comment by richtabor** [comment]
Indeed! I'm glad I could help out. :) 

Here's a [Sketch zip](https://github.com/ahmadawais/WPGulp/files/1860376/wpgulp-logo.zip) and an [SVG zip](https://github.com/ahmadawais/WPGulp/files/1860386/wpgulp-logo.svg.zip).

**Comment by ahmadawais** [comment]
Thanks, everyone! The logo is done.
